### DISCUSSION TITLE
Is there any sensitive information stored in info.plist?

### DISCUSSION THREAD

**Discussion by danstreeter-es** [blacs30/bitwarden-alfred-workflow#149] [Q&A] [answered]
Hey!
First of all - thanks for a great workflow for Alfred... 
BitWarden is awesome, but the thing that was annoying was the rather clunky Chrome integration of lots of mouse use and clicking - this now solves that and my hands no longer leave the keyboard to get passwords! =) THANK YOU!!!

My use case here is probably quite a unique one... maybe not - but hey, here it goes:
I have two machines, both with Alfred installed on them, with their workflows synced between them using Dropbox - works beautifully as expected and all configs and things are synced over...
In the case of BitWarden - my work and personal macs have different logins so I _dont_ want them synced...

I've got around this with a convoluted symlink setup to make `info.plist` in the Alfred workflow directory symlink out to a local directory on the machine, which itself is then symlinked back to my user directory on that machine to be synced/backed up elsewhere.
This gets around the annoyance that my username on each machine being different so I could not use `~/` (the relative home dir which expands out to differing `/Users/username` and `/Users/user.name` paths.)

At present... and I will be looking for an alternative home at some point - I've put this 'pivot-symlink' in `/Users/Shared/.../` which is a static path to both machines and therefore the `/workflow_dir/info.plist` file symlinks to `/Users/Shared/.../info.plist` which is then magically back in the home dir on each respective machine....

So... my question is - is there anything specifically sensitive in that `info.plist` I should be concerned with, or is it being in the 'Shared' user directory actually not that bad...

Alternatively - If anyone has any better ideas for 'Single Alfred sync, double config' in line with my use case above, I'd be welcome to hear about it!

Thanks

**Comment by blacs30** [comment, marked as answer]
Same here, I use the keyboard mostly and don't have the BW browser extensions installed.
I think i would exclude the Bitwarden workflow from syncing in Dropbox, probably the easiest solution.
There are no sensitive information besides the email address stored in cleartext in the workflow dir (should probably move it also into keychain)

**Reply by danstreeter** [reply]
Great, thank you for confirming =)
